content/items/pyvhdlparser.md

```diff
@@ -1,18 +1,39 @@
 ---
 title: "pyVHDLParser"
-description: "A token-stream based parser for VHDL-2008"
+description: "This is a token-stream based parser for VHDL-2008 creating a document object model (DOM)."
```

Comment:
The description is shown as the subtitle. Hence, I think that "*This is*" is not required. Up to you.

Reply:
I'll change it.